modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/AbstractGoCodegen.java

```diff
@@ -486,13 +486,10 @@ public String toEnumVarName(String name, String datatype) {
         if ("int".equals(datatype) || "int32".equals(datatype) || "int64".equals(datatype)
                 || "uint".equals(datatype) || "uint32".equals(datatype) || "uint64".equals(datatype)
                 || "float".equals(datatype) || "float32".equals(datatype) || "float64".equals(datatype)) {
-            String varName = name;
```

Comment:
Golang identifiers can't be prefixed with _*. See fix generated output https://github.com/swagger-api/swagger-codegen/pull/12076/files#diff-e07213988c61a6f77400109f74d3bf2a72aab72499ee220bb02329be26e6e06aL17